src/token.ts

```diff
@@ -120,6 +119,20 @@ export const STAR: Token = {token: 'op', str: '*'};
 export const IMMEDIATE: Token = {token: 'op', str: '#'};
 export const ASSIGN: Token = {token: 'op', str: '='};
 
+// CS -> CS token alias map
+export const CS_TOKEN_ALIAS_MAP = new Map<string, string>([
+  // NOTE: Only synonymous so long as 16-bit is not supported
+  ['.addr', '.word'],
+  // NOTE: Only synonymous so long as js65's .bankbyte differs from ca65's
```

Comment:
If we add this as an option, we'll need to ensure this does the right thing.  The tokenizer will need some state and will need to be a little smarter (also see above).  Probably worth thinking about sooner rather than later.